doc/GuideForCommitters.md

```diff
@@ -26,6 +26,21 @@ follow when reviewing and merging pull requests and issues.  It also provides
 a checklist of items that committers must ensure have been completed prior to
 merging a pull request.
 
+## Committers and Areas of Expertise
+
+The following is an up to date list of the current committers within the project and their respective areas of expertise. It is encouraged to use this list as a guideline to request reviews on your pull requests from committers who is knowledgable in the area.
+
+| Committer          | Username       | Area of Expertise                               |
+|--------------------|----------------|-------------------------------------------------|
+| Andrew Young       | @youngar       | [cmake](https://github.com/eclipse/omr/labels/cmake), [jitbuilder](https://github.com/eclipse/omr/labels/jitbuilder), [port](https://github.com/eclipse/omr/labels/port)                         |
```

Comment:
I would remove JitBuilder from Andrew and add GC.  I am not a leader on the compiler component so I am not sure if Andrew merging JitBuilder PRs without another reviewer makes sense but I will leave that up to others.  Andrew was originally nominated for his work on GC, port and the autotools build system.  Andrew has proven himself as a strong leader on the CMake work so I think that makes a lot of sense to me.

Reply:
Fixed in f52e03d